### ISSUE TITLE
Tukui problem?

### ISSUE THREAD

**Issue by EnochTheWise** [layday/instawow#62]
Hi, today there was a problem updating

```
$ instawow update
✗ tukui:elvui
  package does not exist
$ instawow --version
instawow, version 1.19.0
```

```
$ instawow list
curse:acp
curse:altoholic-retail
curse:atlasloot-enhanced
curse:auctionator
curse:azeroth-auto-pilot
curse:capping-bg-timers
curse:championcommander
curse:deadly-boss-mods
curse:details
curse:elvui-shadow-light
curse:garrisoncommander
curse:gathermate2
curse:heal-bot-continued
curse:healbot-plugin-threat
curse:healbot-plugin-timetodie
curse:healbot-plugin-timetolive
curse:kuinameplates
curse:leatrix-plus
curse:mapster
curse:master-plan
curse:mik-scrolling-battle-text
curse:omni-cc
curse:orderhallcommander
curse:plater-nameplates
curse:rarescanner
curse:recount
curse:scrap
curse:tomtom
curse:war-plan
curse:weakauras-2
instawow:weakauras-companion-autoupdate
tukui:elvui
wowi:23875-blizzmove
```

**Comment by layday** [comment]
Thanks for the report.  I've made some changes to how Tukui add-ons are handled after Tukui dropped the `addon` endpoint (without warning, if I might add), meaning that the ID-less slug isn't recognised for ElvUI anymore.  This might've not been such a great idea in retrospect.  If you don't want to wait for a fix, you can reinstall ElvUI using its Tukui-provided ID:

```sh
$ instawow remove tukui:elvui && instawow install tukui:-2
```

**Comment by layday** [comment]
This has been fixed in 1.19.1.  Apologies for the breakage.
